pack/fhv/fhvc.json

```diff
@@ -118,9 +118,349 @@
+    "back_name": "The Wise Mound",
+    "back_text": "Spawn the set-aside Chelydran Hybrid enemy at the North Shore location. Choose a location adjacent to Chelydran Hybrid's location and spawn 1 of the set-aside Grasping Tendril enemies at that location.\n Shuffle each other set-aside Grasping Tendril enemy into the encounter deck along with the discard pile.",
+    "code": "10591",
+    "clues": 3,
+    "double_sided": true,
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 4,
+    "faction_code": "mythos",
+    "flavor": "The bog is supposedly the origin of the sample that drew Dr. Marquez to the island in the first place. You have your work cut out for you.",
+    "illustrator": "Rob Laskey",
+    "name": "A Botanical Survey",
+    "pack_code": "fhvc",
+    "position": 91,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Objective</b> – At the end of the round, investigators at the North Shore location may spend the requisite number of clues, as a group, to advance. <i>See the Location Placement diagram on page 28 of the campaign guide for the North Short location.)</i>",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "You corner the reptilian creature in a shining pool, where it sinks into the mud, clearly exhausted. As it starts munching on a tangle of flowers, you take several sketches and notes of the thing's sprouting shell. The alkaloid blossom on its back seems to pulse. Producing a pair of shears, you prepare to take a sample.",
+    "back_name": "Strange Anatomy",
+    "back_text": "If it is <b>Day 1</b>, read the following. Otherwise, <b>(→R1)</b>.<blockquote>\"Stop!\" Dr. Marquez shouts. \"That flower is parasitic. It is now part of the creature's anatomy. There's no telling how deeply it's embedded.\" She points to the flower's thick translucent stem: vicious white fluid pumps through it like an artery. \"If the flower dies, its host dies too.\"</blockquote>\n<b>(→R2)</b> <i>(page 30)</i>",
+    "code": "10592",
+    "clues": null,
+    "double_sided": true,
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 5,
+    "faction_code": "mythos",
+    "flavor": "The large flowering turtle-like creature is on the move. Perhaps you can herd it to safety.",
+    "illustrator": "Rob Laskey",
+    "name": "Discovery of a Lifetime",
+    "pack_code": "fhvc",
+    "position": 92,
+    "quantity": 1,
+    "stage": 2,
+    "text": "[fast] Investigators at a single location adjacent to Chelydran Hybrid's location spend 1[player] clues, as a group: For the remainder of the round, change Chelydran Hybrid's patrol location to that location.\n<b>Forced</b> – At the end of the enemy phase: Chelydran Hybrid takes 1 damage for each ready [[Abomination]] enemy at its location.\n<b>Objective</b> – At the end of the round, if the Chelydran Hybrid is at the Starting Location, immediately advance.",
+    "type_code": "act"
+  },
+  {
+    "back_clues": null,
```

Comment:
So for all locations in this set, please can you split out front and back into two cards. An example of that you can find here, in Carnevale, card 6:
https://github.com/Kamalisk/arkhamdb-json-data/blob/master/pack/side/coh_encounter.json

So for the 'front' of these locations, which is the Bog side, you'll delete the 'back' lines and add 

`"back_link": 10593b,`

You also delete `"double_sided": true,"`

Then you create a new entry which is the Sunken side, as follows:

`"code": "10593b",
"encounter_code": "the_thing_in_the_depths",
"encounter_position": 6,
"faction_code": "mythos",
"flavor": "The calm ripples hide something sinister deep below.",
"hidden": true,
"illustrator": "Pixoloid Studios",   
"name": "Open Water",
"pack_code": fhvc,
"position": 6,
"quantity": 1,
"shroud": 5,
"traits": "Sunken.",
"text": "<b>Forced – After you move into this location: Draw the top card of the encounter deck.",
"type_code": "location",
"victory": 1`

For these, you'll need to adjust `code`, `encounter_position`, `position` and delete the `victory` line and final comma, where appropriate.